Tasks/IISWebAppDeployment/MsDeployOnTargetMachines.ps1

```diff
@@ -185,8 +185,10 @@ function Does-WebSiteExists
     $command = "`"$appCmdPath`" $appCmdArgs"
     Write-Verbose "Checking webSite exists. Running Command : $command"
     
+    $ErrorActionPreference = 'Continue'
     $webSite = cmd.exe /c "`"$command`""
```

Comment:
In other PR, you were handling cases for Win10 when it comes to cmd.exe execution. Are we ok here?